aicsimageio/formats.py

```diff
@@ -339,6 +339,7 @@
         "aicsimageio.readers.bioformats_reader.BioformatsReader",
         "aicsimageio.readers.default_reader.DefaultReader",
     ],
+    "omezarr": ["aicsimageio.readers.ome_zarr_reader.OmeZarrReader"],
```

Comment:
I think we need to add `zarr` and maybe `ome.zarr` to this list? (These keys are path suffixes)

Reply:
adding ome.zarr makes sense to me. Should we add zarr if the only support for dimension reading currently comes from metadata with the ome schema (no guess function)?

Reply:
Yes, I think so - the idea being that many people may have made .zarr datasets, some of which will be ome, and some not.  Our "is file supported" could fail gracefully if the .zarr is not using ome-zarr standard.

Reply:
added in [b709d5b](https://github.com/AllenCellModeling/aicsimageio/pull/493/commits/b709d5bc0b284217103cb7813d0bc9937bb48d0d)